### PR TITLE
[0.14.0] Adding diagrams and signposting to each config section

### PR DESCRIPTION
**Pull request by MelHopper**
Signed-off-by: MelHopper <melanieh@uk.ibm.com>

## What type of PR is this ? 

- [ ] Bug fix
- [x ] Enhancement

## What does this PR do ?
Adds diagrams for the Codewind configs to the gettingstarted-overview topic

## Which issue(s) does this PR fix ?
2589

#### Link to the [Codewind repository](https://github.com/eclipse/codewind/issues) issue(s) this PR fixes or references:

https://github.com/eclipse/codewind/issues/2589
## Does this PR require a documentation change ?
Yes - as defined in issue

## Any special notes for your reviewer ?
no

### CODE REVIEW

**Comment by MelHopper** [comment]
@micgibso ready to review. Please can you take a special look at: 

_Continue to learn about each of these configurations by selecting the relevant section:_

and offer other suggestions for how to word, if needed. Many Thanks.

**Comment by MelHopper** [comment]
Note: Callout boxes also added to this Getting Started - Overview topic (gettingstarted-overview.md)

**Comment by micgibso** [comment]
NB: Broken link PR for issue https://github.com/eclipse/codewind/issues/3173 merged after this was created.

File: docs/_documentations/gettingstarted-overview.md [hunk 1]
```diff
@@ -65,13 +75,6 @@ To get started with using Codewind, choose from one of the following configurati
...
-- [Installing Codewind for VS Code](./vsc-getting-started.html)
-- [Installing Codewind for Eclipse](./eclipse-getting-started.html)
-- [Installing Codewind for intellij](./intellij-getting-started.html)
-
-Then, choose from the following options:
-  1. Deploy Codewind to your cloud if not already done so by you or a sysadmin or a DevOps engineer by following instructions to [Deploy Codewind Remotely](./remote-deploying-codewind.html).
-  2. Connect your Codewind extension of your local desktop IDE to Codewind in your cloud by following instructions for [Using Codewind Remotely](./remote-codewind-overview.html). 
-
+<div class="callout">
+	<b>Note:</b> To use Codewind remotely, you must first install Codewind locally. For more information, see instructions for <a href="./remote-deploying-codewind.html"> deploying Codewind Remotely</a> . After Codewind is deployed remotely, you can connect your Codewind extension of your local desktop IDE to Codewind in your cloud by following instructions for <a href="./remote-codewind-overview.html">using Codewind remotely.</a>
```
Comment (micgibso) [inline docs/_documentations/gettingstarted-overview.md:79]: First link in the paragraph has a space after the </a> before the full stop, so remove space.  

Last link in this paragraph, the period is inside the link text, would be better outside, for example: 
<a href="./remote-codewind-overview.html">using Codewind remotely</a>.

File: docs/_documentations/gettingstarted-overview.md [hunk 2]
```diff
@@ -7,55 +7,65 @@ duration: 10 minutes
 permalink: gettingstarted-overview.html
 type: document
 ---
-# Overview: Getting Started
+# Getting Started - Overview
 
-**Getting started quickly with Codewind** You can try out Codewind by using the local configuration. For local use of Codewind, and also the pre-requisite steps for remote use of Codewind, select from:
+<div class="callout">
+	<b>Getting started quickly with Codewind</b> You can try out Codewind by using the local configuration. For local use of Codewind, and also the prerequisite steps for remote use of Codewind, select from <a href="./vsc-getting-started.html">VS Code,</a> <a href="./eclipse-getting-started.html">Eclipse,</a> or <a href="./intellij-getting-started.html">Intellij.</a>
```
Comment (micgibso) [inline docs/_documentations/gettingstarted-overview.md:13]: Last closing `</a>` is outside period, would be better before it.

File: docs/_documentations/gettingstarted-overview.md [hunk 3]
```diff
@@ -7,55 +7,65 @@ duration: 10 minutes
...
+</div>
 
-* [VS Code](./vsc-getting-started.html)
-* [Eclipse](./eclipse-getting-started.html)
-* [IntelliJ](./intellij-getting-started.html)
-
-**Codewind as a hosted application in the cloud** If you want to go straight to using codewind as a hosted application in the cloud, select the [Eclipse Che instructions](./eclipseche-codewind-overview.html)
+<div class="callout">
+	<b>Browser-based Codewind</b> If you want to go straight to using codewind as a hosted application in the cloud, select the <a href="./eclipseche-codewind-overview.html">Eclipse Che instructions. </a>
```
Comment (micgibso) [inline docs/_documentations/gettingstarted-overview.md:17]: Last closing `</a>` is outside period, would be better before it.

File: docs/_documentations/gettingstarted-overview.md [hunk 4]
```diff
@@ -7,55 +7,65 @@ duration: 10 minutes
...
 Codewind supports different configurations for developing your containerized applications, so you can use Codewind with whichever development flow you choose.
 
 You can do your development work in an IDE on your desktop, or in an IDE hosted on the cloud.
 
-You can choose to build and run your application locally, or use the cloud's resources by using Codewind remotely or as an application hosted on the cloud. Codewind uses [Kubernetes](https://kubernetes.io/) container orchestration applications, including [OpenShift](https://www.openshift.com/) when building and run your Docker-containerised applications.
+You can choose to build and run your application locally, or build and run your application using the cloud's resources. 
+
+Both the remote configuration and browser-based configuration of Codewind use the cloud's resources to build and run your application. Codewind uses [Kubernetes](https://kubernetes.io/) container orchestration applications, including [OpenShift](https://www.openshift.com/), to build and run your Docker-containerised applications on the cloud.
+
+Continue to learn about each of these configurations by selecting the relevant section:
```
Comment (micgibso) [inline docs/_documentations/gettingstarted-overview.md:30]: I like: 
"Continue to learn more about each of these configurations by clicking the following options:"

**Pull request merged**